SCClassLibrary/Common/Quarks/Quarks.sc

```diff
@@ -9,7 +9,8 @@ Quarks {
 		fetched=false,
 		cache,
 		regex,
-		installedPaths;
+		installedPaths,
+		<>installing;
```

Comment:
Were these getters/setters just for debugging? do they need to be exposed?

Reply:
good catch - yes I'll remove